src/eventcontroller/EventName.ts

```diff
@@ -19,6 +19,7 @@ type EventName =
   | 'receivingAudioDropped'
   | 'sendingAudioFailed'
   | 'sendingAudioRecovered'
-  | 'backgroundFilterConfigSelected';
+  | 'backgroundFilterConfigSelected'
+  | 'deviceLabelTriggerFailed';
```

Comment:
`deviceLabelTriggerFailed` may not be very accurate. For example there're two cases here to cause the failure:
1. camera, microphone device permission has been denied before join the meeting, in this case the device trigger is not triggered or (prompt does not show up).
2. end user denies the device permission in the pop up prompt.

Do we need to relate this event with deviceLabelTrigger? How about device permission acquisition failure?

For React SDK we have these states, just for reference.
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/main/src/types/index.ts#L69-L74

```ts
export enum DeviceLabelTriggerStatus {
  UNTRIGGERED = 'UNTRIGGERED',
  IN_PROGRESS = 'IN_PROGRESS',
  GRANTED = 'GRANTED',
  DENIED = 'DENIED',
}
```

Reply:
For both (1) and (2), the SDK invokes the `deviceLabelTrigger` function.

https://github.com/aws/amazon-chime-sdk-js/blob/20f2e1aa4c813ec1a885f581da0cb79c556d62c8/src/devicecontroller/DefaultDeviceController.ts#L824-L839



Naming this event presents some challenges for the following reasons:
- Subsequent functions like `enumerateDevices` may also generate an error.
- A developer could supply a custom function, such as performing an operation unrelated to device permissions, as explained here.
   https://github.com/aws/amazon-chime-sdk-js/blob/20f2e1aa4c813ec1a885f581da0cb79c556d62c8/src/devicecontroller/DeviceControllerFacade.ts#L25-L42


Given that our public API name `deviceLabelTrigger` is prone to causing an error, I have simply named it according to this API name.